### DISCUSSION TITLE
Есть ли смысл добавлять "state_template"!?!?

### DISCUSSION THREAD

**Discussion by Sergey-SRG** [dext0r/yandex_smart_home#435] [Ideas]
Есть ли возможность добавить "state_template", так сказать с заделом на будущее!?!?!?
Что бы не плодить темплейных сущностей. Было бы очень здорово и полезно, думаю не только мне ;)

**Comment by dext0r** [comment]
Реализовано в мастере (https://docs.yaha-cloud.ru/master/advanced/capabilities/about/#state)
